File: 1.Arrays/Kotlin/RemoveDuplicateFromSorted.kt
fun main() {
    var arr = arrayOf(12, 15, 15, 29, 37)
    var count = 0

    for (i in 0..4) {
        if (arr[i] != arr[count]) {
            arr[count++] = arr[i]
        }
    }

    // If want zero in remaining places
    while (count < 5) {
        arr[count++] = 0
    }

    for (i in 0..count - 1) {
        print("${arr[i]} ")
    }
}

File: 1.Arrays/C/RemoveDuplicateFromSorted.c
#include <stdio.h>

int main() {
    int arr[] = {12, 15, 15, 29, 37};
    int count = 0;

    for (int i = 0; i < 5; i++) {
        if (arr[i] != arr[count]) {
            arr[count++] = arr[i];
        }
    }

    // Fill remaining places with zero
    while (count < 5) {
        arr[count++] = 0;
    }

    // Print the array
    for (int i = 0; i < count; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}